GenericsHomework.Tests/NodeTests.cs

```diff
@@ -0,0 +1,358 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace GenericsHomework.Tests
+{
+
+    [TestClass]
+    public class NodeTests
+    {
+
+        public string ToString(int[] array) => "{" + string.Join(", ", array) + "}";
+        public string ToString(ArrayList list) => "{" + string.Join(", ", list.ToArray()) + "}";
+        public string ToString<T>(List<T> list) => "{" + string.Join(", ", list.ToArray()) + "}";
+
+
+        public Node<int> CreateInt4NodeList(int num1, int num2, int num3, int num4)

```

Comment:
```suggestion
        public static Node<int> CreateInt4NodeList(int num1, int num2, int num3, int num4)
```
Also can be marked as static, the same goes for the methods below.